CHANGELOG.md

```diff
@@ -678,3 +678,4 @@ Initial version to have a stable reference before [gossip protocol removal](/doc
 - Bucket commands.
 - Downsampling support for UI.
 - Grafana dashboards for Thanos components.
+- Failint for imported packages verification.
```

Comment:
I think we should not touch 0.1.0 release  ;p

Also this is a tooling, not really relevant for Thanos users, so we don't need changelog entry. (:

Reply:
I was following the guide :) Removed!

Reply:
About not touching the 0.1.0 version, should I change the PR to a different branch instead of master?

Reply:
No. Master is fine. Just you put the changelog entry at the bottom, but actually releases goes from newest to oldest.

Anyway, there should be no entry.